client/pages/configuration/general.vue

```diff
@@ -1,3 +1,19 @@
+<script setup lang="ts">
+const config = storeToRefs(useGlobalConfiguration())
+const { t } = useI18n()
+
+const toggleRegistrations = computed(() => config.toggleRegistrations.value)
+const defaultDiskQuotaPolicy = computed(() => config.defaultDiskQuotaPolicy.value)
+const defaultDiskQuota = computed(() => config.defaultDiskQuota.value)
```

Comment:
```suggestion
const config = useGlobalConfiguration()
const { t } = useI18n()

const toggleRegistrations = computed(() => config.toggleRegistrations)
const defaultDiskQuotaPolicy = computed(() => config.defaultDiskQuotaPolicy)
const defaultDiskQuota = computed(() => config.defaultDiskQuota)
```

(you can get even get rid of the computed properties altogether tbh)

Reply:
I wanted it to be reactive in the first place, but this requires to use a model instead of a regular prop. Maybe for later...

Reply:
`config` *IS* reactive ! `storeToRefs`'s only purpose is to convert the reactive like object to and object of refs. That is to say, if you're not destructuring the return value of `storeToRefs`, there is absolutely no point in using it: that is its only purpose.